pkg/binlog/position_test.go

```diff
@@ -73,3 +73,92 @@ func (t *testPositionSuite) TestPositionFromStr(c *C) {
 		c.Assert(pos, DeepEquals, cs.pos)
 	}
 }
+
+func (t *testPositionSuite) TestRealMySQLPos(c *C) {
+	cases := []struct {
+		pos       gmysql.Position
+		expect    gmysql.Position
+		errMsgReg string
+	}{
+		{
+			pos:    gmysql.Position{Name: "mysql-bin.000001", Pos: 154},
+			expect: gmysql.Position{Name: "mysql-bin.000001", Pos: 154},
+		},
+		{
+			pos:    gmysql.Position{Name: "mysql-bin|000002.000003", Pos: 154},
```

Comment:
we can set log-bin = mysql|bin, and then the binlog file will looks like `mysql|bin.000001`, need add a test case like `mysql|bin|000002.000003`

Reply:
Oh! any advice about differing real pos and UUID added pos with the same format of `mysql|uuid.000001`?

Reply:
added some simple verify for UUID suffix and test cases in https://github.com/pingcap/dm/pull/140/commits/1011afbb43e73b598eff807955182a107c55907c.
but we still can't handle all cases well.

Reply:
how about define a new struct `Position` like:
```
type Position struct {
    Name string
    Pos int64
    UUID string // or UUID suffix
}
```

Reply:
but seems still can not handle the case `log-bin=mysql|000001`?